Tests/LibWeb/Text/expected/form-formEnctype-attribute.txt

```diff
@@ -0,0 +1,103 @@
+button: unset
+button.getAttribute('formEnctype') == 'null'
+button.formEnctype == 'application/x-www-form-urlencoded'
```

Comment:
This is wrong, the correct value is an empty string

Reply:
This might be due to https://github.com/LadybirdBrowser/ladybird/blob/master/Userland/Libraries/LibWeb/HTML/HTMLFormElement.cpp#L452-L471